src/ui/input/color-picker/color-picker.spec.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { PreviewColorPicker, DisabledColorPicker } from './color-picker.composition';
+
+it('should render with text, and L size', () => {
+  const { container, getByText } = render(<PreviewColorPicker />);
+  const inputElement = container.querySelector('input');
+  const text = getByText('Select color');
+
+  expect(inputElement).not.toBeNull();
+  expect(inputElement?.disabled).toEqual(false);
+  expect(inputElement?.checked).toEqual(false);
```

Comment:
I don't think you need to test `.checked`

Reply:
I would test that selecting a color calls `onInputChanged`. Not critical

Reply:
I copied the test from another input component lol.
I removed this line from the test.